src/system/process.py

```diff
@@ -12,12 +12,14 @@
 
 
 class Process:
-    def __init__(self) -> None:
+    def __init__(self, filename: str, distribution: str) -> None:
```

Comment:
Process is a generic class in `system`, it should have nothing to do with distribution, this doesn't belong here.